booking-engine-api/deprecations/README.md

```diff
@@ -28,3 +28,5 @@ The table columns have the following meanings:
 | [Configuration](../operations/configuration.md#configuration-1) | `PaymentCardInput` | `PaymentCardRequirement` | [Get configuration](../operations/configuration.md#get-configuration) | 18 Apr 2021 | - |
 | [Room category](../operations/hotels.md#room-category) | `ImageIds` | [Category image assignment](../operations/configuration.md#category-image-assignment) \([Get configuration](../operations/configuration.md#get-configuration)\) | [Get configuration](../operations/configuration.md#get-configuration), [Get hotels](../operations/hotels.md#get-hotels) | 21 Oct 2021 | - |
 | [Total Amount](../operations/reservations.md#totalamount) | `TaxValues` | [Breakdown](../operations/reservations.md#totalamount) | [Get reservation price](../operations/reservations.md#get-reservation-price) | 19 Oct 2022 | - |
+| [Adult Count](../operations/reservations#reservation) | `AdultCount` | [OccupancyData](../operations/reservations#occupancy-data) | [Get reservation price](../operations/reservations.md#get-reservation-price) | 1 Jan 2024 | - |
```

Comment:
The deprecation applies to more than just **Get reservation price**, we should list all the affected operations. It's a bit of a chore, but the purpose of the table is to make it easy for users to know if a particular deprecation affects them or not.